contracts/proxy/GeneralTransferManagerProxy.sol

```diff
@@ -26,6 +26,7 @@ contract GeneralTransferManagerProxy is GeneralTransferManagerStorage, ModuleSto
     {
         require(_implementation != address(0), "Implementation address should not be 0x");
         _upgradeTo(_version, _implementation);
+        _setDefaults();
```

Comment:
Wonder whether this should be in the GTM initialisation function instead? Possibly in a future upgraded GTM this function may change.

Reply:
I guess it only gets called the first time the proxy is created and the logic contract is set (not on future upgradeToAndCall calls) so this is fine.